app/views/user.py

```diff
@@ -408,13 +424,22 @@ def put(self, user_type_id):
         try:
             # retrieve body data from input JSON
             data = request.get_json()
-            user_type = UserType.query.filter(UserType.id == user_type_id).first()
-            if not user_type:
+            errors = user_type_schema.validate(data, partial=True)
+            if errors:
```

Comment:
Print `errors` and also send `errors` in the response so that by looking at the response we will understand what schema validations are filaing.